webapp/package.json

```diff
@@ -11,6 +11,7 @@
     "bootstrap-colorpicker": "2.5.1",
     "chart.js": "2.5.0",
     "compass-mixins": "0.12.10",
+    "exif2css": "^1.2.0",
```

Comment:
Could you lock this to `1.2.0` instead of using `^1.2.0`? It might not be necessary any more now that we've switched to yarn, but it'll keep it consistent with our other versioning